src/compas_eve/__init__.py

```diff
@@ -66,4 +72,7 @@
     "set_default_transport",
     "InMemoryTransport",
 ]
+
```

Comment:
Remove all importing of ZeroMQTransport directly into the main `__init__`, this should only be an explicit import action by the user